packages/atomic/stencil.config.ts

```diff
@@ -40,7 +40,13 @@ export const config: Config = {
     {
       type: 'dist',
       esmLoaderPath: '../loader',
-      copy: [{src: 'themes'}],
+      copy: [
+        {src: 'themes'},
+        {
+          src: '../node_modules/coveo-styleguide/resources/icons/svg/coveo-search-ui-filetypes',
```

Comment:
That works but it's not perfect.

When I look at react-vapor repo, these icons all seems to be fixed in times (16 months ago), and not imported/refreshed from Salesforce design system from time to time.

In an ideal world, I think we would actually want to depend from Salesforce design system directly, if possible.

Reply:
🤔  I'll see what I can do